File: android/app/src/main/java/com/nhs/online/nhsonline/data/PaycassoExternalReferences.kt
package com.nhs.online.nhsonline.data

import com.squareup.moshi.JsonClass

@JsonClass(generateAdapter = true)
data class PaycassoExternalReferences (
    val consumerReference: String,
    val transactionReference: String,
    val appUserId: String,
    val deviceId: String,
    val hasNfcJourney: Boolean,
    val transactionType: String
)

File: xamarin/NHSOnline.App.iOS/DependencyServices/PaycassoExternalReferences.cs
namespace NHSOnline.App.iOS.DependencyServices
{
    public sealed class PaycassoExternalReferences
    {
        public string ConsumerReference { get; set; } = string.Empty;
        public string TransactionReference { get; set; } = string.Empty;
        public string AppUserId { get; set; } = string.Empty;
        public string DeviceId { get; set; } = string.Empty;
        public string TransactionType { get; set; } = string.Empty;
        public bool HasNfcJourney { get; set; } = false;
    }
}